exchanges/okex/okex_wrapper.go

```diff
@@ -378,27 +456,33 @@ func (o *OKEX) UpdateTicker(p currency.Pair, assetType asset.Item) (*ticker.Pric
 			return nil, err
 		}
 
+		enabled, err := o.GetEnabledPairs(asset.Futures)
```

Comment:
Thanks for making those changes! I'm happy with things. Worst part now is that we have no ability to disable all pairs for a given asset type. So having OKEX enabled means we will perpetually log:
```
Failed to get REST ticker. Error: ticker fetching not supported for index
Failed to get REST orderbook. Error: no orderbooks for index
```
I think disabling an asset type of pairs in the config is out of scope and something for future work, however is there something that can be done about constantly logging the above?

Reply:
I actually I would like that included, so I will add an update in. I also would like to upgrade GRPC to enable/disable pairs on the fly. @thrasher- Can I add these two to the PR or separate them?

Reply:
Can include it in this PR. We already support modifying enabled/disabled pairs via gRPC but not as a whole by a asset type, so that will be a nice feature

Reply:
Perfect I will get onto it 👍